client/Routes.js

```diff
@@ -1,40 +1,72 @@
 import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
 import { withRouter, Route, Switch, Redirect } from 'react-router-dom';
-import { Login, Signup } from './components/AuthForm';
+import { Login, Signup } from './components/AuthForm.js';
 import Home from './components/Home';
 import AllProducts from './components/AllProducts';
 import SingleProduct from './components/SingleProduct';
+import Brands from './components/Brands';
+import OrderHistory from './components/OrderHistory';
+import ProductReviews from './components/ProductReviews';
+import CreateReview from './components/CreateReview';
+import UpdateProduct from './components/UpdateProduct';
+import Cart from './components/Cart';
+import NotFoundPage from './components/NotFoundPage';
 import { me } from './store';
+import { fetchProducts } from "./store/products";
+import { fetchCart } from "./store/cart";
 
 /**
  * COMPONENT
  */
 class Routes extends Component {
   componentDidMount() {
     this.props.loadInitialData();
+    this.props.fetchProducts();
```

Comment:
Because the line above this is called `loadInitialData`, the dispatches could all be moved into a single function in your `mapDispatchToProps`, so that "loading the initial data" loads all of the data for your component (whatever that initial data may be)